[4.1] Обновление до новой версии + новые расширения

Kostelano opened the pull request:
Работа над переводом текущих изменений для 4.1 + новых расширений.

Joomla 4.1 запланирована на 15 февраля.

Kostelano committed:
Joomla 4.1 RC1

https://github.com/joomla/joomla-cms/pull/30522 +
https://github.com/joomla/joomla-cms/pull/32223 +
https://github.com/joomla/joomla-cms/pull/31675 +
https://github.com/joomla/joomla-cms/pull/35378 +
https://github.com/joomla/joomla-cms/pull/35612 +
https://github.com/joomla/joomla-cms/pull/35715 +
https://github.com/joomla/joomla-cms/pull/35610 +
https://github.com/joomla/joomla-cms/pull/35607 +
https://github.com/joomla/joomla-cms/pull/35788 +
https://github.com/joomla/joomla-cms/pull/35647 +
https://github.com/joomla/joomla-cms/pull/35143 +
https://github.com/joomla/joomla-cms/pull/36135 +
https://github.com/joomla/joomla-cms/pull/35998 +
https://github.com/joomla/joomla-cms/pull/36173 +
https://github.com/joomla/joomla-cms/pull/36212 +
https://github.com/joomla/joomla-cms/pull/36208 +
https://github.com/joomla/joomla-cms/pull/36206 +
https://github.com/joomla/joomla-cms/pull/36205 +
https://github.com/joomla/joomla-cms/pull/36203 +
https://github.com/joomla/joomla-cms/pull/36192 +
https://github.com/joomla/joomla-cms/pull/36191 +
https://github.com/joomla/joomla-cms/pull/36228 +
https://github.com/joomla/joomla-cms/pull/36211 +
https://github.com/joomla/joomla-cms/pull/36271 +
https://github.com/joomla/joomla-cms/pull/36270 +
https://github.com/joomla/joomla-cms/pull/36245 +
https://github.com/joomla/joomla-cms/pull/36294 +
https://github.com/joomla/joomla-cms/pull/36244 +
https://github.com/joomla/joomla-cms/pull/36242 +
https://github.com/joomla/joomla-cms/pull/36296 +
https://github.com/joomla/joomla-cms/pull/36190 +
https://github.com/joomla/joomla-cms/pull/36474 +
https://github.com/joomla/joomla-cms/pull/36297 +
https://github.com/joomla/joomla-cms/pull/36480 +
https://github.com/joomla/joomla-cms/pull/36479 +
https://github.com/joomla/joomla-cms/pull/36551 +
https://github.com/joomla/joomla-cms/pull/36366 +
https://github.com/joomla/joomla-cms/pull/36589 +
https://github.com/joomla/joomla-cms/pull/36583 +
https://github.com/joomla/joomla-cms/pull/36328 +
https://github.com/joomla/joomla-cms/pull/36515 +
https://github.com/joomla/joomla-cms/pull/36555 +
https://github.com/joomla/joomla-cms/pull/36653 +
https://github.com/joomla/joomla-cms/pull/36660 +
https://github.com/joomla/joomla-cms/pull/36657 +
https://github.com/joomla/joomla-cms/pull/36637 +
https://github.com/joomla/joomla-cms/pull/35983 +
https://github.com/joomla/joomla-cms/pull/36704 +
https://github.com/joomla/joomla-cms/pull/36708 +
https://github.com/joomla/joomla-cms/pull/36700 +

Kostelano committed:
Новое в Joomla 4.1 RC + исправления (1)

https://github.com/joomla/joomla-cms/pull/36835 +
https://github.com/joomla/joomla-cms/pull/36180 +
https://github.com/joomla/joomla-cms/pull/36721 +
https://github.com/joomla/joomla-cms/pull/36735 +
https://github.com/joomla/joomla-cms/pull/36739 +
https://github.com/joomla/joomla-cms/pull/36740 +
https://github.com/joomla/joomla-cms/pull/36848 +

Kostelano committed:
Версия в XML

Kostelano committed:
Фикс кавычек в plg_system_updatenotification

b2z commented on administrator/language/ru-RU/com_config.ini:
```diff
@@ -95,8 +104,9 @@ COM_CONFIG_FIELD_LOG_CATEGORY_MODE_EXCLUDE="Исключить"
 COM_CONFIG_FIELD_LOG_CATEGORY_MODE_INCLUDE="Включить"
 COM_CONFIG_FIELD_LOG_CATEGORY_MODE_LABEL="Режим категории логов"
 COM_CONFIG_FIELD_LOG_DEPRECATED_LABEL="Логировать устаревший API"
+COM_CONFIG_FIELD_LOG_EVERYTHING_DESC="Включить или отключить функцию логирования событий, за&#160;исключением категории устаревшего API."
```
Что-то тут вкралось `&#160;`
`за&#160;исключением`

b2z replied:
Тут не исправлено

Kostelano replied:
https://github.com/JPathRu/localisation/commit/853e11599953daa294ed17c2f7cc130fe529a966

Это не ошибка и не опечатка, а неразрывный пробел. Частицы в/не/на/за и т.д. лучше не переносить на новую строку отдельно от слова. Но т.к. мониторы у всех разные - не угадаешь перенос строки, можно попробовать этим пренебречь.

b2z replied:
Ах вот оно что. Ну если есть такая практика, может всё таки оставим её. Я просто не знал. Хотя вижу, что ты уже убрал...

Kostelano replied:
По админке много где используется неразрывный пробел. Скажем, на моем мониторе (он небольшой) перенос строки как раз после "за" - я это замечаю, ставлю неразрыв. Но мониторов куча, разрешений куча, все не отследишь и на 100% не исправишь. Не стршно :)

b2z commented on administrator/language/ru-RU/com_scheduler.ini:
```diff
@@ -0,0 +1,162 @@
+COM_SCHEDULER_MSG_MANAGE_NO_TASK_PLUGINS="Ничего не найдено."
+COM_SCHEDULER_NEW_TASK="Задача"
+COM_SCHEDULER_N_ITEMS_DELETED_1="%d задача успешно удалена."
+COM_SCHEDULER_N_ITEMS_DELETED_2="%d задачи успешно удалено."
```
%d задачи успешно удален**ы**

И дальше по тексту везде _2 изменить на **ы**

Kostelano replied:
Исправлено в https://github.com/JPathRu/localisation/commit/cd08854ddd9c446c44047ee62b134c8067c34ccb

b2z commented on administrator/language/ru-RU/com_scheduler.ini:
```diff
@@ -0,0 +1,162 @@
+COM_SCHEDULER_TOOLBAR_UNLOCK="Разблокировать"
+COM_SCHEDULER_TYPE_CHOOSE="Выбор типа задачи"
+COM_SCHEDULER_WARNING_EXISTING_TASK_TYPE_NOT_FOUND="Не удалось найти сценарий для выполнения задачи. Вероятно, плагин планировщика задач отключен или удален."
+COM_SCHEDULER_XML_DESCRIPTION="Компонент для управления задачами."
```
"Компонент для управления планировщиком задач."

Kostelano replied:
Я вот думаю. Компонент является планировщиком задач - он так и называется, и управлять самим собой так себе формулировка. Да и в других компонентах плюс/минус идентичная представленной формулировка.

```
COM_BANNERS_XML_DESCRIPTION="Компонент для управления баннерами и клиентами."
COM_CACHE_XML_DESCRIPTION="Компонент для управления группами кэша."
COM_CATEGORIES_XML_DESCRIPTION="Компонент для управления категориями."
COM_CONTACT_XML_DESCRIPTION="Компонент для управления контактами."
COM_CONTENT_XML_DESCRIPTION="Компонент для управления материалами."
```

И так далее. Что скажешь?

b2z replied:
Меня тут беспокоит только то, что Управление задачами не до конца раскрывает, о каких задачах идёт речь. Слишком абстрактно, задачи разные бывают. Что-то пока не могу придумать как лучше обыграть.

progreccor replied:
планировщик задач нормальный термин. Навряд ли вы найдете что-то лучше

Kostelano replied:
Исправлено в https://github.com/JPathRu/localisation/pull/84/commits/ef6c3889fe7c24904a70fd5c2df9928d3145d9d4

b2z commented on administrator/language/ru-RU/com_scheduler.sys.ini:
```diff
@@ -0,0 +1,16 @@
+COM_SCHEDULER_MSG_MANAGE_NO_TASK_PLUGINS="Ничего не найдено."
+COM_SCHEDULER_TASKS_VIEW_DEFAULT_DESC="Отображение списка задач для выполнения"
+COM_SCHEDULER_TASKS_VIEW_DEFAULT_TITLE="Планировщик задач"
+COM_SCHEDULER_XML_DESCRIPTION="Компонент для управления задачами."
```
"Компонент для управления планировщиком задач."

Kostelano replied:
Исправлено в https://github.com/JPathRu/localisation/pull/84/commits/ef6c3889fe7c24904a70fd5c2df9928d3145d9d4

b2z commented on administrator/language/ru-RU/com_tags.ini:
```diff
@@ -104,6 +104,13 @@ COM_TAGS_N_ITEMS_TRASHED="%d тегов успешно перемещено в 
+COM_TAGS_N_QUICKICON="Тегов"
+COM_TAGS_N_QUICKICON_SRONLY_0="Теги: Нет опубликованных тегов"
+COM_TAGS_N_QUICKICON_SRONLY_1="Теги: %d тег опубликован"
+COM_TAGS_N_QUICKICON_SRONLY_2="Теги: %d тега опубликовано"
```
опубликован**ы**

Kostelano replied:
Исправлено в https://github.com/JPathRu/localisation/commit/cd08854ddd9c446c44047ee62b134c8067c34ccb

b2z commented on administrator/language/ru-RU/plg_task_demotasks.ini:
```diff
@@ -0,0 +1,19 @@
+; Note : This file needs to be saved as UTF-8
+
+PLG_TASK_DEMO_TASKS="Задача - Демонстрация"
+PLG_TASK_DEMO_TASKS_SLEEP_TIMEOUT_LABEL="Тайм-аут сна (секунд)"
```
Я бы писал (секунды)

Kostelano replied:
В других строках локализации мы используем **X (минут)**, **X (часов)**, **X (дней)**. Я бы предпочел не менять на "секунды", иначе есть смысл менять везде :)

b2z replied:
Это моё мнение. Я бы писал в именительном падеже. В компоненте что-то я это пропустил.

b2z commented on administrator/language/ru-RU/plg_task_demotasks.ini:
```diff
@@ -0,0 +1,19 @@
+PLG_TASK_DEMO_TASKS_RESUMABLE_TITLE="Возобновляемая задача"
+PLG_TASK_DEMO_TASKS_RESUMABLE_DESC="Задача для демонстрации возобновляемого поведения."
+PLG_TASK_DEMO_TASKS_RESUMABLE_STEPS_LABEL="Количество шагов"
+PLG_TASK_DEMO_TASKS_RESUMABLE_TIMEOUT_LABEL="Тайм-аут на шаг (секунд)"
```
Я бы писал (секунды)

b2z commented:
@Kostelano в plg_user_token отсутствует перевод

`PLG_USER_TOKEN_TOKEN_DESC="This is your token. Use it in third party applications requiring remote access to your site through the Joomla API application."`
Предлагаю такой перевод
`Это ваш токен. Используйте его в сторонних приложениях, требующих удаленного доступа к вашему сайту через приложение Joomla API.`

b2z commented:
@Kostelano ты запросил ревью, но не вижу изменений.

Kostelano committed:
Фикс плюрализации _2

Kostelano commented:
> В plg_user_token отсутствует перевод
> 
> `PLG_USER_TOKEN_TOKEN_DESC="This is your token. Use it in third party applications requiring remote access to your site through the Joomla API application."` Предлагаю такой перевод `Это ваш токен. Используйте его в сторонних приложениях, требующих удаленного доступа к вашему сайту через приложение Joomla API.`

Упс, пропустил еще при переводе 4.0.
Исправлено в https://github.com/JPathRu/localisation/commit/cd08854ddd9c446c44047ee62b134c8067c34ccb.

Kostelano committed:
Удалил неразрывный пробел

Kostelano committed:
Фикс плюрализации _2 (2)

* Окончание "о", когда а/я
аутентификатора, баннера, изменения, клиента, ключа, контакта, материала, модуля, перенаправления,
переопределения, перехода, плагина, поля, пользователя, процесса, пункта меню, расширения, сервера,
согласия, сообщения, стиля, тега, типа, уровня, файла, фильтра, элемента, этапа, языка

* Окончание "ы", когда и/ы
версии, группы, задачи, заметки, записи, карты, категории, ленты, проблемы

Kostelano committed:
Еще правки

Kostelano committed:
Еще правки (2)

Kostelano committed:
Еще правки (3)

b2z commented:
@Kostelano ты закончил, можно делать ревью?

Kostelano commented:
Да, все залил
